test/e2e/run_transient_test.go

```diff
@@ -63,7 +63,7 @@ var _ = Describe("Podman run with volumes", func() {
 		session.WaitWithDefaultTimeout()
 		Expect(session).Should(Exit(0))
 
-		// All files should be in permament store, volatile
+		// All files should not be in permanent store, not volatile
```

Comment:
Now I think there's an extra `not`?

Reply:
I think they are both needed.
All files should not be in the permanent store, because the program --rm (removed) them.
Also this is not on a volatile store.

Reply:
I'm having a lot of trouble parsing the comment... but that's beyond the scope of this PR.